apps/api_accounts/lib/api_accounts.ex

```diff
@@ -658,6 +658,8 @@ defmodule ApiAccounts do
   end
 
   def totp_uri(%User{} = user) do
-    NimbleTOTP.otpauth_uri("MBTA:#{user.email}", user.totp_secret_bin, issuer: "MBTA")
+    NimbleTOTP.otpauth_uri("MBTA-API: api-v3.mbta.com/#{user.email}", user.totp_secret_bin,
```

Comment:
I'm curious where your seeing `HOST` defined? At least locally, `System.get_env("HOST") => nil`. I'm not sure where the Kubernetes manifests are, but unless it's being defined in there I'd expect `HOST` to either be undefined or the hostname of the machine rather than the hostname of the site.

Reply:
The best way I'd know how to get it is the `HOST` header, so I'd have to pass that into `totp_uri` from the controller.

Reply:
AHA! Thanks. I'll use that then. The nil locally shouldn't cause an issue, it'll just be missing the hostname.

Reply:
and pushed

Reply:
Done